### DISCUSSION TITLE
Tshock Not working

### DISCUSSION THREAD

**Discussion by shosho12344** [Pryaxis/TShock#1707] [General]
* TShock version: 4.3.8

1. I first downloaded Tshock version 4.3.8
2. Extracted the folder
3. Ran the Terraria Tshock server
4. This is what it showed me

```
[Server API] Error Server crashed due to an unhandled exception:
System.ArgumentNullException: Value cannot be null.
Parameter name: path
   at System.IO.Path.GetFullPathInternal(String path)
   at System.IO.File.InternalGetLastWriteTimeUtc(String path, Boolean checkHost)
   at System.IO.File.GetLastWriteTime(String path)
   at Terraria.Main.DedServ()
   at Terraria.ProgramServer.InnerStart(String[] args)
```
SCREENSHOT: 

![Capture]
https://user-images.githubusercontent.com/64290809/80265410-5894ba80-864c-11ea-847a-92d0bde49850.PNG

**Comment by moisterrific** [comment]
Why are you using 4.3.8? That’s an older version meant for version 1.3.0.8 of the game.

The latest release compatible with PC Terraria 1.3.5.3 without startup issues is 4.3.26, you can find it under [releases](https://github.com/Pryaxis/TShock/releases/tag/v4.3.26).

**Comment by shosho12344** [comment]
> Why are you using 4.3.8? That’s an older version meant for version 1.3.0.8 of the game.
> 
> The latest release compatible with PC Terraria 1.3.5.3 without startup issues is 4.3.26, you can find it under [releases](https://github.com/Pryaxis/TShock/releases/tag/v4.3.26).

Because I wanna host a mobile Tshock server. People have done it so its not impossible.

**Comment by QuiCM** [comment]
Unfortunately we don't support mobile Terraria.
If you're using the Terraria configuration file or commandline arguments when starting the server, ensure the values are correct. Also note that any world files created after 1.3.0.8 may cause issues when the server tries to load them

**Comment by shosho12344** [comment]
>when starting the server, ensure the values are correct. 

What do you mean by this?

**Comment by moisterrific** [comment]
If you want to host a mobile server feel free to get the released mobile port off my GitHub or from chi-rei-den. Though I will most likely not be keeping it further updated when mobile goes past 1.3.0.7.

**Comment by shosho12344** [comment]
> If you want to host a mobile server feel free to get the released mobile port off my GitHub or from chi-rei-den. Though I will most likely not be keeping it further updated when mobile goes past 1.3.0.7.

Can this get commands in game?

**Comment by moisterrific** [comment]
> > If you want to host a mobile server feel free to get the released mobile port off my GitHub or from chi-rei-den. Though I will most likely not be keeping it further updated when mobile goes past 1.3.0.7.
> 
> Can this get commands in game?

Yes, all the commands are working except for the ones relating to 1.3.5.3 PC content which mobile does not have, such as NPC party mode, the Old One’s Army, etc.

**Comment by shosho12344** [comment]
> > > If you want to host a mobile server feel free to get the released mobile port off my GitHub or from chi-rei-den. Though I will most likely not be keeping it further updated when mobile goes past 1.3.0.7.
> > 
> > 
> > Can this get commands in game?
> 
> Yes, all the commands are working except for the ones relating to 1.3.5.3 PC content which mobile does not have, such as NPC party mode, the Old One’s Army, etc.

Do you have a youtube video showing how to do this? I am kinda lost, sorry

**Comment by moisterrific** [comment]
> > > > If you want to host a mobile server feel free to get the released mobile port off my GitHub or from chi-rei-den. Though I will most likely not be keeping it further updated when mobile goes past 1.3.0.7.
> > > 
> > > 
> > > Can this get commands in game?
> > 
> > 
> > Yes, all the commands are working except for the ones relating to 1.3.5.3 PC content which mobile does not have, such as NPC party mode, the Old One’s Army, etc.
> 
> Do you have a youtube video showing how to do this? I am kinda lost, sorry

Nope

**Comment by shosho12344** [comment]
> > > > > If you want to host a mobile server feel free to get the released mobile port off my GitHub or from chi-rei-den. Though I will most likely not be keeping it further updated when mobile goes past 1.3.0.7.
> > > > 
> > > > 
> > > > Can this get commands in game?
> > > 
> > > 
> > > Yes, all the commands are working except for the ones relating to 1.3.5.3 PC content which mobile does not have, such as NPC party mode, the Old One’s Army, etc.
> > 
> > 
> > Do you have a youtube video showing how to do this? I am kinda lost, sorry
> 
> Nope

dang it

**Comment by hakusaro** [comment]
This has been converted to a discussion, which means that it now looks weird! Hooray! To change this back to the way it looked before, change the sorting to “chronological” instead of “most relevant” on the side of the page, or on the bottom if you’re viewing this on mobile.
